CHANGELOG.md

```diff
@@ -4,6 +4,7 @@
 
 :new: What's new:
 - Update lakeFS version to [1.54.0](https://github.com/treeverse/lakeFS/releases/tag/v1.54.0)
+- For enterprise: added support for multi resource policies 
```

Comment:
1. don't reuse the changelog it's already a new version 1.4.10
2. in the new changelog: Update fluffy version to 0.11.0 (added support for multi resource policies)